golang/go-cloud-run-hello-world/CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+Join us at https://github.com/GoogleCloudPlatform/cloud-code-samples.
+
+## Run the Tests
```

Comment:
Same comment as in nodejs - this info should be at the root CONTRIBUTING.md

Reply:
This has sample-specific instructions that will be even less generic once we centralize cloudbuild.yaml by language. I guess we could have a CONTRIBUTING per language and list per sample instructions?

Reply:
yep, that was the intent

Reply:
yes